.github/workflows/format.yml

```
@@ -38,7 +38,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install Python dependencies
-        run: pip install --constraint requirements-dev.txt black pylint isort
+        run: pip install --group dev pylint isort black
```

```suggestion
        run: pip install --group dev
```
These three are already specified in the `dev` group.